### PR TITLE
HW-58096. SDK for transfer refunds.

### PR DESCRIPTION
**Pull request by kkobziev-hw**

### CODE REVIEW

**Review by Rrathinasabapath — APPROVED** [review]
Verified the changes. Ready to be reviewed by Aparna.

**Review by arao6 — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,8937 @@
+{
```
Comment (vkorotchenko) [inline package-lock.json:1]: We usually dont push the lock files. is there a reason we are pushing this in?

File: .gitignore [hunk 2]
```diff
@@ -32,3 +32,5 @@ lib/
 
 # Docs folder
 docs/
+
+.nyc_output/
```
Comment (vkorotchenko) [inline .gitignore:36]: where is this file generated?

**Review by vkorotchenko — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (Ssangaran) [reply package-lock.json]: removed lock file

File: .gitignore (hunk 2 above)
Comment (Ssangaran) [reply .gitignore]: removed .nyc_output/

**Pull request merged**